tests/qos/test_qos_sai.py

```diff
@@ -1923,7 +1960,8 @@ def testQosSaiDwrrWeightChange(
     @pytest.mark.parametrize("queueProfile", ["wm_q_wm_all_ports"])
     def testQosSaiQWatermarkAllPorts(
         self, queueProfile, ptfhost, dutTestParams, dutConfig, dutQosConfig,
-        resetWatermark, _skip_watermark_multi_DUT, skip_pacific_dst_asic
+        get_src_dst_asic_and_duts, resetWatermark, _skip_watermark_multi_DUT,
+        skip_pacific_dst_asic
```

Comment:
why add cisco specific platform info to generic case?
can you change this name more generic?

Reply:
QWMAllPorts is a cisco-specific test as you can see from this conditional marker:
<img width="1170" alt="Screenshot 2023-10-31 at 9 47 38 AM" src="https://github.com/sonic-net/sonic-mgmt/assets/58446052/51a4e03d-df0c-46eb-9594-76923fccfbc0">